packages/adapter-node/index.js

```diff
@@ -32,8 +32,10 @@ export default function (opts = {}) {
 
 			if (precompress) {
 				builder.log.minor('Compressing assets');
-				await builder.compress(`${out}/client`);
-				await builder.compress(`${out}/prerendered`);
+				await Promise.allSettled([
```

Comment:
wouldn't it be better to use `Promise.all` so that if one were to fail for some reason then we'd get the failure immediately? or is there some benefit to using `allSettled`?

Reply:
Sorry, actually I added `allSettled` thinking that code needs to be executed even after any one of the promises failed. 

Yh if it is needed to thrown error preferred to change to `all` 🤝

Reply:
> wouldn't it be better to use `Promise.all` so that if one were to fail for some reason then we'd get the failure immediately? or is there some benefit to using `allSettled`?

Done :)